### PR TITLE
CASMPET-5341 1.0 : Skeleton DOC procedures for Add Remove Replace NCN

### PR DESCRIPTION
**Pull request by kimjensen-hpe**
## Summary and Scope

Adds alpha framework doc for Add, Remove, Replace NCNs

## Issues and Related PRs

* Resolves [CASMPET-5341](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5341)
* Change will also be needed in NA
* Future work required by [CASM-2769](https://jira-pro.its.hpecorp.net:8443/browse/CASM-2769)
* Documentation changes required in NA
* Merge with/before/after NA

## Testing

NA

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by rgross727 — APPROVED** [review]

**Comment by nrockershousen** [comment]
We should add these new files to the operations index.md file: https://github.com/Cray-HPE/docs-csm/blob/release/1.2/operations/index.md#node-management

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Alpha Framework to Add, Remove or Replace NCNs
+
+- [Add Worker, Storage or Master NCNs](#add-worker-storage-master)
+- [Remove Worker, Storage or Master NCNs](#remove-worker-storage-master)
+- [Replace Worker, Storage or Master NCNs](#replace-worker-storage-master)
+
+<a name="add-worker-storage-master"></a>
+# Add Worker, Storage or Master NCNs
```
Comment (cdelatte-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:8]: I don't think you can have a # top level followed by another # top level.

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+6. [Update Firmware](Add_Remove_Replace_NCNs/Update_Firmware.md) via FAS
+
+7. [Boot NCN](Add_Remove_Replace_NCNs/Boot_NCN.md)
+
+8. [Validation](Add_Remove_Replace_NCNs/Validation.md#validate-added-ncn)
+
+
+<a name="remove-worker-storage-master"></a>
+# Remove Worker, Storage or Master NCNs
```
Comment (cdelatte-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:49]: Same top level issue like commented above

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+5. [Update Networking](Add_Remove_Replace_NCNs/Update_Networking.md#update-networking-to-remove-ncn)
+
+6. [Validate SHCD](Add_Remove_Replace_NCNs/Validate_SHCD.md#validate-shcd-after-removing-ncn)
+
+7. [Validation](Add_Remove_Replace_NCNs/Validation.md#validate-removed-ncn)
+
+
+<a name="replace-worker-storage-master"></a>
+# Replace Worker, Storage or Master NCNs
```
Comment (cdelatte-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:77]: top level same as above

**Review by cdelatte-hpe — CHANGES_REQUESTED** [review]

**Review by bklei — APPROVED** [review]

**Review by cdelatte-hpe — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

**Pull request merged**